gadget-default.Dockerfile

```diff
@@ -56,7 +61,7 @@ RUN set -ex; \
 # - https://github.com/kinvolk/traceloop/actions
 # - https://hub.docker.com/r/kinvolk/traceloop/tags
 
-FROM docker.io/kinvolk/traceloop:20211109004128958575 as traceloop
+FROM docker.io/kinvolk/traceloop:2059b729c0ac8aa79016dacb06fbdaa1867c1446 as traceloop
```

Comment:
What's changing from traceloop?

Reply:
This image is cross-built, the previous one not.
But I think I can drop this change.